bundles/org.openhab.binding.hdpowerview/src/main/resources/OH-INF/thing/shade.xml

```diff
@@ -49,6 +50,7 @@
 		</supported-bridge-type-refs>
 		<label>PowerView Shade</label>
 		<description>Hunter Douglas (Luxaflex) PowerView Gen3 Shade</description>
+		<semantic-equipment-tag>WindowCovering</semantic-equipment-tag>
```

Comment:
Is there a reason for having `Blinds` for Gen 1/2 and `WindowCovering` for Gen 3?

Reply:
No. I think 95% of the time all equipment will be Blinds. However HD does have motorised Drapes/Curtains in their range. So probably all should offer Blinds. (User can obviously override if they want..) ??